integration/terraform/ec2/linux/variables.tf

```diff
@@ -0,0 +1,72 @@
+variable "ec2_instance_type" {
+  type = string
+  default = "t3a.xlarge"
+}
+
+variable "key_name" {
+  type = string
+  default = "cwagent-integ-test-key"
+}
+
+variable "iam_instance_profile" {
+  type = string
+  default = "CloudWatchAgentServerRole"
+}
+
+variable "vpc_security_group_ids" {
+  type = list(string)
+  default = ["sg-013585129c1f92bf0"]
+}
+
+variable "region" {
+  type = string
+  default = "us-west-2"
+}
+
+variable "aws_access_key" {
+  type = string
+}
+
+variable "aws_secret_access_key" {
+  type = string
+}
+
+variable "ami" {
+  type = string
+  default = ""
+}
+
+variable "ssh_key" {
+  type = string
+  default = ""
+}
+
+variable "user" {
+  type = string
+  default = ""
+}
+
+variable "github_sha" {
+  type = string
+  default = ""
+}
+
+variable "github_repo" {
+  type = string
+  default = ""
+}
+
+variable "package" {
+  type = string
+  default = ""
+}
+
+variable "install_agent" {
```

Comment:
Similar to `package, `install_agent` feels ambiguous for what the value should be here. I think this warrants a `description` value in the variable declaration.